### ISSUE TITLE
Today's dev version has displays left dark

### ISSUE THREAD

**Issue by petepawn64** [flybywiresim/a32nx#4874]
### Aircraft Version

Development

### Build info

```json
"built": "2021-05-23T14:09:52+00:00",
  "ref": "master",
  "sha": "9d1a3198703fa4d6554353104201899aba7f6605",
  "actor": "lukecologne",
  "event_name": "manual"
```

### Describe the bug

Everything at the startup working but all displays remain dark

### Expected behavior

As it was yesterday

### Steps to reproduce

Start

### Discord Username (optional)

petepawn

**Comment by sidnov** [comment]
It's a known issue and is being worked on at the moment

**Comment by fahdriyami** [comment]
Good to knows it's being worked on. Thought it was just me. I also attempted uninstalling and reinstalling. Same issue. Looking forward to the fix!

**Comment by AndreasTremml** [comment]
There are also 0.0 V on the batteries and even with external power or APU the voltage doesn't increase! There is just power for basic instrumentation and lighting, but not for aviation, ATC, etc.

**Comment by Benjozork** [comment]
@AndreasTremml we are aware and it's being worked on..

**Comment by sidnov** [comment]
Fixed with #4750
